x-pack/plugin/security/qa/multi-cluster/build.gradle

```diff
@@ -5,9 +5,27 @@
  * 2.0.
  */
 
+import org.elasticsearch.gradle.internal.info.BuildParams
+import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
+
 apply plugin: 'elasticsearch.internal-java-rest-test'
 apply plugin: 'elasticsearch.rest-resources'
+apply plugin: 'elasticsearch.bwc-test'
 
 dependencies {
   clusterModules(project(":modules:analysis-common"))
 }
+
+tasks.named("javaRestTest") {
+  enabled = true
+  // This is tested explicitly in bwc test tasks.
+  exclude '**/RemoteClusterSecurityBwcRestIT.class'
+}
+
+BuildParams.bwcVersions.withWireCompatible(v -> v.before('8.8.0')) { bwcVersion, baseName ->
+  tasks.register(bwcTaskName(bwcVersion), StandaloneRestIntegTestTask) {
+    usesBwcDistribution(bwcVersion)
+    systemProperty("tests.old_cluster_version", bwcVersion)
+    include '**/RemoteClusterSecurityBwcRestIT.class'
```

Comment:
One (potential) improvement could be to define dedicated package where we would put all BWC tests, then exclude all classes from it. This way we would avoid the need to include every class individually.